zppy/templates/ts.bash

```diff
@@ -147,19 +147,39 @@ EOF
   cp -s $dest/*_{{ '%04d' % (yr_start) }}??_{{ '%04d' % (yr_end) }}??.nc $input_dir
   dest_cmip={{ output }}/post/{{ component }}/{{ grid }}/cmip_ts/{{ frequency }}
   mkdir -p ${dest_cmip}
+
   {{ e3sm_to_cmip_environment_commands }}
+
+  {% if input_files.split(".")[0] == 'cam' or input_files.split(".")[0] == 'eam' -%}
+  #add code to do vertical interpolation variables at model levels before e3sm_to_cmip
+  IFS=',' read -ra mlvars <<< "{{ interp_vars }}"
+  for var in "${mlvars[@]}"
+  do
+    for file in ${input_dir}/${var}_{{ '%04d' % (yr_start) }}??_{{ '%04d' % (yr_end) }}??.nc
+    do
+      if [ -f ${file} ]; then
+        #ncks --rgr xtr_mth=mss_val --vrt_fl='{{cmip_plevdata}}' ${file} ${file}.plev
+        ncremap -p mpi --vrt_ntp=log --vrt_xtr=mss_val --vrt_out='{{cmip_plevdata}}' ${file} ${file}.plev
```

Comment:
It looks like this block uses NCO rather than `e3sm_to_cmip` and should thus be moved before the change of environment (`{{ e3sm_to_cmip_environment_commands }}`)

Reply:
We should also look at implementing https://github.com/E3SM-Project/zppy/issues/467 (as a separate PR).

Reply:
Include vertical remapping with the already-implemented horizontal remapping?

Reply:
We should also check if vertical remapping is already covered by `ncclimo`